src/test/java/org/broadinstitute/dsm/ElasticSearchTest.java

```diff
@@ -684,29 +686,70 @@ public void createTestParticipantsInES() throws Exception {
+            String[] includes = new String[] {object};
+            String[] excludes = Strings.EMPTY_ARRAY;
+            FetchSourceContext fetchSourceContext = new FetchSourceContext(true, includes, excludes);
+            GetRequest getRequest = new GetRequest()
+                    .index(index)
+                    .type("_doc")
+                    .id(ddpParticipantId)
+                    .fetchSourceContext(fetchSourceContext);
+            GetResponse getResponse = client.get(getRequest, RequestOptions.DEFAULT);
+            return getResponse.getSourceAsMap();
+        }
+    }
 
-            Map<String, Object> workflowMap = new HashMap<>();
-            workflowMap.put("workflow", "ACCEPTANCE_STATUS");
-            workflowMap.put("status", "ACCEPTED");
-
-            List<Map<String, Object>> workflowList = new ArrayList<>();
-            workflowList.add(workflowMap);
+    @Test
+    public void updateWorkflowValues() throws Exception {
+        Map<String, Object> workflowMapES = getObjectByID("participants_structured.rgp.rgp", "XLDUNC3BHGWGWERHW781", "workflows");
+        if (workflowMapES != null && !workflowMapES.isEmpty()) {
+            List<Map<String, Object>> workflowListES = (List<Map<String, Object>>) workflowMapES.get("workflows");
+            if (workflowListES != null && !workflowListES.isEmpty()) {
+                boolean updated = false;
+                for (Map<String, Object> worklist : workflowListES) {
+                    if ("ALIVE_DECEASED".equals(worklist.get("workflow"))) {
+                        //update value in existing workflow
+                        worklist.put("status", "ALIVE");
+                        worklist.put("date", SystemUtil.getISO8601DateString());
+                        updated = true;
+                    }
+                }
+                if (!updated) {
+                    Map<String, Object> newWorkflowMap = new HashMap<>();
+                    newWorkflowMap.put("workflow", "ALIVE_DECEASED");
+                    newWorkflowMap.put("status", "ALIVE");
+                    newWorkflowMap.put("date", SystemUtil.getISO8601DateString());
+                    workflowListES.add(newWorkflowMap);
+                }
+            }
+            else {
+                //add workflow
+                Map<String, Object> newWorkflowMap = new HashMap<>();
+                newWorkflowMap.put("workflow", "ALIVE_DECEASED");
+                newWorkflowMap.put("status", "ALIVE");
+                newWorkflowMap.put("date", SystemUtil.getISO8601DateString());
+                workflowListES.add(newWorkflowMap);
+                workflowMapES = new HashMap<>();
+                workflowMapES.put("workflows", workflowListES);
+            }
+        }
+        updateES("participants_structured.rgp.rgp", "XLDUNC3BHGWGWERHW781", workflowMapES);
+        Map<String, Object> workflowMapES2 = getObjectByID("participants_structured.rgp.rgp", "XLDUNC3BHGWGWERHW781", "workflows");
 
```

Comment:
I guess, you wanted to use assertion here like it was using searchProfileValue before?